storage/client_split_test.go

```diff
@@ -359,6 +359,10 @@ func TestStoreZoneUpdateAndRangeSplit(t *testing.T) {
 	store := createTestStore(t)
 	defer store.Stop()
 
+	maxBytes := int64(1 << 16)
+	rng := store.LookupRange(engine.KeyMin, nil)
+	fillRange(store, rng.Desc().RaftID, proto.Key("test"), maxBytes, t)
```

Comment:
It's subtle that this test continues to test what it was previously testing; namely, that the range is split due to exceeding RangeMaxBytes. You might add a comment at line 376 explaining that the put of the new zone config is what causes Range.maybeSplit to enqueue the range for splitting.